grpcclient/grpcclient.go

```diff
@@ -37,6 +37,9 @@ type Config struct {
 	// https://github.com/grpc/grpc/blob/master/doc/connection-backoff.md
 	ConnectBackoffBaseDelay time.Duration `yaml:"connect_backoff_base_delay" category:"advanced"`
 	ConnectBackoffMaxDelay  time.Duration `yaml:"connect_backoff_max_delay" category:"advanced"`
+
+	GRPCMiddleware       []grpc.UnaryClientInterceptor  `yaml:"-"`
+	GRPCStreamMiddleware []grpc.StreamClientInterceptor `yaml:"-"`
```

Comment:
We could also call it `Middleware` to avoid repeating `GRPCClientConfig.GRPCMiddleware`.

Reply:
I think calling it `Middleware` makes sense since this is already a `grpc*` package but I don't feel strongly about it.